From 92efe875b104200d70b43d74099273872efc109d Mon Sep 17 00:00:00 2001
From: Anish Trivedi <anish@freescale.com>
Date: Wed, 13 Apr 2011 14:18:45 -0500
Subject: [PATCH 0107/2463] ENGR00142089-2 MX51/MX53 Upgrade SCC2 and SAHARA
 drivers to 2.6.38

Removed SCC2 IRQ definitions from header file since
IRQs will be obtained from resource array during probe
function.

Signed-off-by: Anish Trivedi <anish@freescale.com>
---
 include/linux/mxc_scc2_driver.h |   10 ----------
 1 files changed, 0 insertions(+), 10 deletions(-)

diff --git a/include/linux/mxc_scc2_driver.h b/include/linux/mxc_scc2_driver.h
index 4a32d62..91327e4 100644
--- a/include/linux/mxc_scc2_driver.h
+++ b/include/linux/mxc_scc2_driver.h
@@ -119,16 +119,6 @@
 #define SCC_DRIVER_MINOR_VERSION_2    2
 
 
-/*!
- *  Interrupt line number of SCM interrupt.
- */
-#define INT_SCC_SCM         MXC_INT_SCC_SCM
-
-/*!
- *  Interrupt line number of the SMN interrupt.
- */
-#define INT_SCC_SMN         MXC_INT_SCC_SMN
-
 /**
  * @typedef scc_return_t
  */
-- 
1.7.7.4

